emojivoto-web-app/js/setup_dev_env.ps1

```diff
@@ -0,0 +1,209 @@
+        $demo_cluster_url = "https://staging-auth.datawire.io/api/democlusters/telepresence-demo/config"
+    }
+
+    Invoke-WebRequest $demo_cluster_url -Headers @{ "X-Ambassador-API-Key" = $env:AMBASSADOR_API_KEY } -ContentType "application/json" -OutFile 'emojivoto_k8s_context.yaml'
+    $env:KUBECONFIG = './emojivoto_k8s_context.yaml'
+    kubectl config set-context --current --namespace=emojivoto 2>&1 | Out-Null
+
+    Write-Host "Listing services in ${Global:EMOJIVOTO_NS} namespace"
+    $listSVC = kubectl --namespace ${Global:EMOJIVOTO_NS} get svc
+    $listSVC
+    send_telemetry "connectedToK8S"
+}
+
+function install_upgrade_telepresence{
+    display_step(3)
+    $install_telepresence = $false
+    Write-Host -NoNewline 'Checking for Telepresence ... '
+    if (!(Get-Command telepresence -ErrorAction SilentlyContinue)) {
+        $install_telepresence = $true
+        Write-Host "Installing Telepresence"
+    } else {
+        # Ensure that running telepresence daemons are stopped. A running daemon
+        # has its current working directory set to the directory where it was first
+        # started, and since the KUBECONFIG is set to a relative directory in this
+        # script, a previously started daemon might resolve it incorrectly.
+        telepresence quit -s > $null
+        if ((telepresence version) -match "upgrade") {
+            $install_telepresence = $true
+            Write-Host "Upgrading Telepresence"
+        } else {
+            Write-Host "Telepresence already installed"
+            send_telemetry("telepresenceAlreadyInstalled")
+        }
+    }
+    if ($install_telepresence) {
+        $telepresence_download_url = "https://app.getambassador.io/download/tel2/windows/amd64/2.13.3/telepresence-setup.exe"
+        Invoke-WebRequest $telepresence_download_url -OutFile telepresence-setup.exe
+        Start-Process .\telepresence-setup.exe -NoNewWindow -Wait
+        Remove-Item telepresence-setup.exe -Recurse -Confirm:$false -Force
+        $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine")
+        send_telemetry("telepresenceInstalled")
+    }
+}
+
+function connect_local_dev_env_to_remote{
+    $env:KUBECONFIG = './emojivoto_k8s_context.yaml'
+    display_step 5
+    Write-Host 'Connecting local development environment to remote K8s cluster'
+
+    $svcName = "edge-stack"
+    telepresence quit 2>&1 | Out-Null
+    telepresence helm upgrade --team-mode 2>&1 | Out-Null
+    telepresence login --apikey="$Env:AMBASSADOR_API_KEY"  2>&1 | Out-Null
+    telepresence quit -s 2>&1 | Out-Null
+    telepresence connect --docker 2>&1 | Out-Null
+
+    $interceptName = (kubectl get rs -n emojivoto --selector=app=web-app --no-headers -o custom-columns=":metadata.name")
+    telepresence intercept "$interceptName" --docker --context default -n "$Global:EMOJIVOTO_NS" --service web-app --port 8083:80 --ingress-port 80 --ingress-host "$svcName.ambassador" --ingress-l5 "$svcName.ambassador"
```

Comment:
This line just hang forever for me

![image](https://github.com/datawire/emojivoto/assets/16439060/54668d89-efa2-4f5a-baea-baca4ab93164)

Reply:
Do you have the issue ? Or is it a me problem ?